packages/mdc-button/README.md

```diff
@@ -41,47 +41,30 @@ npm install --save @material/button
 
 ## Usage
 
-### Button type
-
-> Note: Examples and documents use generic `<button>`, but users can also apply
-`mdc-button` to `<a class="mdc-button">Link Button</a>` in cases where it is
-semantically correct.
-
-#### Text Button
-
+### HTML Structure
 ```html
 <button class="mdc-button">
-  Text button
-</button>
-```
-
-#### Raised Button
-
-```html
-<button class="mdc-button mdc-button--raised">
-  Raised button
+  Button
 </button>
 ```
 
-#### Unelevated Button
-
-```html
-<button class="mdc-button mdc-button--unelevated">
-  Unelevated button
-</button>
-```
+> Note: Examples and documents use generic `<button>`, but users can also apply
+`mdc-button` to `<a class="mdc-button">Link Button</a>` in cases where it is
+semantically correct.
 
-#### Stroked Button
+### CSS Classes
 
-```html
-<button class="mdc-button mdc-button--stroked">
-  Stroked button
-</button>
-```
+| Class                 | Description                                             |
+| --------------------- | ------------------------------------------------------- |
+| `mdc-button`          | A text button that is flush with the surface.           |
```

Comment:
Should this be documented as Mandatory and the others documented as Optional?

Reply:
That's a great question!
IMHO, the html structure have already demonstrated this point, so I might not feel confused. But it should be useful to someone new to this library, I will add it.

Reply:
The reason I mentioned both "Mandatory" and "Optional" is it would match the practice e.g. in the FAB docs: https://github.com/material-components/material-components-web/tree/master/packages/mdc-fab#css-classes